bm_common_structs.h

```diff
@@ -122,6 +122,35 @@ typedef struct {
     uint8_t data[0];
 } __attribute__((packed)) bm_common_wireless_network_data_header_t;
 
+typedef struct {
+  // crc of the data
+  uint32_t crc32;
+  // Data
+  uint8_t data[0];
+} __attribute__((packed)) bm_common_system_configuration_header_t;
```

Comment:
i think system_configuration is not the clearest name since it is used for one of the partitions in memory, but it was what was previously used. Maybe we can use system_info?

Reply:
Yeah I would changes this to bm_network_crc